core/templates/css/oppia.css

```diff
@@ -2543,6 +2543,14 @@ oppia-parameter {
   min-width: 200px;
   width: 400px;
 }
+
+.oppia-modal-lesson-information-card .modal-dialog {
```

Comment:
Any particular reason for adding this css inside oppia.css and not in the component file ?

Reply:
No particular reasons. Just 'modal dialog' and such were original properties of mat-card and not classes. Also, style for another information card was implemented in oppia.css. So, I thought it would it better to add it here.

Reply:
While /deep is deprecated, I worked on the lines of how a similar modal component (oppia-modal-information-card) was implemented.